src/relax/transform/fuse_ops.cc

```diff
@@ -908,23 +908,27 @@ class PatternBasedPartitioner : ExprVisitor {
   using GroupMap = OperatorFusor::GroupMap;
   using ExprVisitor::VisitExpr_;
 
-  static GroupMap Run(String pattern_name, DFPattern pattern, Expr expr, support::Arena* arena) {
-    PatternBasedPartitioner part(pattern_name, pattern, AnalyzeVar2Value(expr));
+  static GroupMap Run(String pattern_name, DFPattern pattern, runtime::PackedFunc check, Expr expr,
+                      support::Arena* arena) {
+    PatternBasedPartitioner part(pattern_name, pattern, check, AnalyzeVar2Value(expr));
     // Initialize each expr to have its own group
     PostOrderVisit(
         expr, [arena, &part](const Expr& e) { part.group_map_[e.get()] = arena->make<Group>(); });
     part.VisitExpr(expr);
     return part.group_map_;
   }
 
-  PatternBasedPartitioner(String pattern_name, DFPattern pattern, const Map<Var, Expr>& bindings)
+  PatternBasedPartitioner(String pattern_name, DFPattern pattern, runtime::PackedFunc check,
```

Comment:
Will it be better to use stronger typing, replacing `PackedFunc` with `TypedPackedFunc`, so that it's clearer on what to pass here?

Reply:
Updated, there are still a few places using `Array<PackedFunc>` because `TypedPackedFunc` can't be Array element